db/re/mob_db.yml

```diff
@@ -82968,120 +82968,2610 @@ Body:
+        Rate: 600
+      - Item: Condor_Card
+        Rate: 1
+        StealProtected: true
+  - Id: 3389
+    AegisName: MIN_WILOW
+    Name: Willow
+    Level: 101
+    Hp: 40000
+    BaseExp: 1300
+    JobExp: 1350
+    Attack: 702
+    Attack2: 946
+    Defense: 151
+    MagicDefense: 2
+    Str: 113
+    Agi: 54
+    Vit: 73
+    Int: 39
+    Dex: 124
+    Luk: 29
+    AttackRange: 1
+    SkillRange: 10
+    ChaseRange: 12
+    Size: Medium
+    Race: Plant
+    Element: Earth
+    ElementLevel: 1
+    WalkSpeed: 200
+    AttackDelay: 1672
+    AttackMotion: 672
+    DamageMotion: 432
+    Ai: 01
+    Modes:
+      Aggressive: true
+    Drops:
+      - Item: Tree_Root
+        Rate: 9000
+      - Item: Wooden_Block
+        Rate: 100
+      - Item: Resin
+        Rate: 2000
+      - Item: Sweet_Potato
+        Rate: 1000
+      - Item: Tree_Of_Archer_3
+        Rate: 3000
+      - Item: Tree_Of_Archer_2
+        Rate: 2000
+      - Item: Tree_Of_Archer_1
+        Rate: 1000
+      - Item: Wilow_Card
+        Rate: 10
+        StealProtected: true
+  - Id: 3390
+    AegisName: MIN_SPORE
+    Name: Spore
+    Level: 101
+    Hp: 40000
+    BaseExp: 1300
+    JobExp: 1350
+    Attack: 703
+    Attack2: 947
+    Defense: 152
+    MagicDefense: 10
+    Str: 114
+    Agi: 55
+    Vit: 74
+    Int: 40
+    Dex: 125
+    Luk: 30
+    AttackRange: 1
+    SkillRange: 10
+    ChaseRange: 12
+    Size: Medium
+    Race: Plant
+    Element: Water
+    ElementLevel: 1
+    WalkSpeed: 200
+    AttackDelay: 1872
+    AttackMotion: 672
+    DamageMotion: 288
+    Ai: 01
+    Modes:
+      Aggressive: true
+    Drops:
+      - Item: Mushroom_Spore
+        Rate: 8000
+      - Item: Red_Herb
+        Rate: 1000
+      - Item: Wing_Of_Fly
+        Rate: 500
+      - Item: Spore_Doll
+        Rate: 10
+      - Item: Hat
+        Rate: 30
+      - Item: Poison_Spore
+        Rate: 10
+      - Item: Strawberry
+        Rate: 600
+      - Item: Spore_Card
+        Rate: 10
+        StealProtected: true
+  - Id: 3391
+    AegisName: MIN_POPORING
+    Name: Poporing
+    Level: 101
+    Hp: 40000
+    BaseExp: 1300
+    JobExp: 1350
+    Attack: 704
+    Attack2: 948
+    Defense: 153
+    MagicDefense: 17
+    Str: 115
+    Agi: 56
+    Vit: 75
+    Int: 41
+    Dex: 126
+    Luk: 31
+    AttackRange: 1
+    SkillRange: 10
+    ChaseRange: 12
+    Size: Medium
+    Race: Plant
+    Element: Poison
+    ElementLevel: 1
+    WalkSpeed: 300
+    AttackDelay: 1672
+    AttackMotion: 672
+    DamageMotion: 480
+    Ai: 02
+    Modes:
+      Aggressive: true
+    Drops:
+      - Item: Sticky_Mucus
+        Rate: 6000
+      - Item: Garlet
+        Rate: 1000
+      - Item: Green_Herb
+        Rate: 1000
+      - Item: Grape
+        Rate: 200
+      - Item: Wing_Of_Fly
+        Rate: 500
+      - Item: Main_Gauche
+        Rate: 20
+      - Item: Apple
+        Rate: 125
+      - Item: Poporing_Card
+        Rate: 10
+        StealProtected: true
+  - Id: 1056
```

Comment:
You're re-defining Smokie here? This is why it's saying the mob drop is over 10.

Reply:
It was a typo, forgot to change his ID and aegisname, done :D